main.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'app'
+
+def show_menu
+  puts ''
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
```

Comment:
- [ ] Aside from this, I noticed that the other options that were to be created as seen in the project requirement are not working as expected and end up throwing errors which in turn break the app so kindly fix these as well :+1:  
- [ ] From the video below you can see that most of the options throw an error. So kindly fix this and ensure that your app works exactly as seen in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)[example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) :100: 

[Screencast from 06-21-2023 02:43:06 PM.webm](https://github.com/Naledi-Dikgale/oop-school-library/assets/102699958/b617830a-91ef-40be-b2e2-187164791896)